core/Cookie.php

```diff
@@ -43,6 +43,11 @@ class Cookie
      */
     protected $path = '';
 
+    /**
+     * @var string
+     */
+    protected $keyStore = false;
```

Comment:
doesn't look like a string!

Reply:
But the property is actually meant to hold a string. Even though the method setting this property has the default `false` 🤷
I actually don't like the way we are handling our variables in many places. We often use to either set a variable to `false` if it's not used and to a string if it has content. In many cases we even check for the `false` and have some comments like `@var false|string`, which is actually not "valid" as `false` is no valid type. But using `bool` isn't correct either, as the value can never be `true`, as that might break the code in some cases.
Imho the correct way to handle that would be using `null` instead of `false`, but I didn't want to do a bigger refactoring as part of this small PR...